src/servers/src/http.rs

```diff
@@ -306,7 +306,8 @@ pub enum GreptimeQueryOutput {
 #[derive(Default, Debug, Clone, Copy, PartialEq, Eq)]
 pub enum ResponseFormat {
     Arrow,
-    Csv,
+    // (with_names, with_types)
+    Csv(bool, bool),
```

Comment:
nitpick: Can we name these fields?
```suggestion
    Csv { with_names: bool, with_types: bool },
```

Reply:
There are only two fields, and the current implementation is appropriate.